### PR TITLE
docs(api): remove phone plugin reference

### PR DESCRIPTION
**Pull request by abailey-dev**
Removes phone plugin references in the SDK docs "calling" section and renames section to "meetings".
Fixes #SPARK-117145

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: documentation/config.yml [hunk 1]
```diff
@@ -5,14 +5,7 @@ toc:
...
   - AuthorizationNode
-  - name: Calling
-  - Phone
-  - PhoneConfig
-  - Call
-  - CallMemberships
-  - CallMembership
-  - StatsStream
-  - StatsFilter
+  - name: Meetings
```
Comment (adamweeks) [inline documentation/config.yml:8]:
```suggestion
  - name: Webex Meetings
    description: |
      The meetings plugin allows browser users to embed Webex meetings into their application.
  
```

**Review by adamweeks — CHANGES_REQUESTED** [review]
Looks like naming the section the same as the plugin is causing issues with double expanding. This seems to fix that issue and gives a little detail on what the section is for.

File: documentation/config.yml (hunk 1 above)
Comment (abailey-dev) [reply documentation/config.yml]: Thanks!

**Comment by adamweeks** [comment]
@abailey-dev you've got a merge conflict, otherwise this looks good!

**Review by adamweeks — APPROVED** [review]

**Pull request merged**